toolchain/check/context.cpp

```diff
@@ -89,11 +89,13 @@ auto Context::DiagnoseNameNotFound(Parse::Node parse_node, StringId name_id)
                  semantics_ir_->strings().Get(name_id));
 }
 
-auto Context::NoteIncompleteClass(SemIR::ClassDeclaration class_decl,
+auto Context::NoteIncompleteClass(SemIR::ClassId class_id,
                                   DiagnosticBuilder& builder) -> void {
   CARBON_DIAGNOSTIC(ClassForwardDeclaredHere, Note,
                     "Class was forward declared here.");
-  builder.Note(class_decl.parse_node, ClassForwardDeclaredHere);
+  auto& class_info = semantics_ir().classes().Get(class_id);
```

Comment:
```suggestion
  const auto& class_info = semantics_ir().classes().Get(class_id);
```

Even though it's a short function, maybe explicit const?

Reply:
Done.